backend/src/handlers/grade.ts

```diff
@@ -15,12 +15,13 @@ import { createAssignment, getStudents, postGrades } from '../util/canvas';
 import { calculate } from './sessionGrades';
 import { Sequelize } from 'sequelize';
 
-const mockUserid = 1;
 
 // GET /api/v1/courses/:courseId/grades
 export const getSessionsGrades = async (
 	event: APIGatewayEvent
 ): Promise<ProxyResult> => {
+
+	const currentUser: any = event.requestContext.authorizer;
```

Comment:
please either make a type or use the `ignore any` thing so we don't have warnings

Reply:
Make a real type please

using eslint ignore for the use of any is just adding more code debt to this project